src/nativeAssetManager.js

```diff
@@ -227,12 +271,45 @@ export function newNativeAssetManager(win) {
     }
 
     if (data.message === 'assetResponse') {
-      const body = win.document.body.innerHTML;
-      const newHtml = replace(body, data);
+      const body = win.document.body.innerHTML,flag = (typeof win.pbNativeData !== 'undefined');
 
-      win.document.body.innerHTML = newHtml;
-      callback && callback();
-      win.removeEventListener('message', replaceAssets);
+      if (flag && data.adId != win.pbNativeData.adId) return;
```

Comment:
Any particular reason to use `!=` vs `!===`?

Do we expect adId to be the same data type in both? Or is is possible one is string and one is number?

Reply:
Nope good catch. Updated